root/release/edit/information.tt

```diff
@@ -314,110 +314,253 @@
+             { doc => { href => doc_link('Style/Release#Status'), target => '_blank' } }) %]
       </p>
     <!-- /ko -->
   </div>
 
-  <div class="bubble" data-bind="bubble: $root.packagingBubble">
+  <div class="bubble" data-bind="bubble: $root.languageBubble">
+    <p>
+      [% l('The {doc|language} attribute should be set to the language used for the release title and track titles.
+            It should not be set to the language the lyrics are written in,
+            nor to the language used for other extra information on the cover.',
+           { doc => { href => doc_link('Release#Language'), target => '_blank' } }) %]
+    </p>
     <p>
-      [% l('Review the {packaging|list of packaging types} for help.',
-           { packaging => { href => doc_link('Release/Packaging'), target => '_blank' } }) %]
+      [% l('Please see the {doc|style guidelines} for more information.',
+           { doc => { href => doc_link('Style/Release#Language_and_script'), target => '_blank' } }) %]
     </p>
   </div>
 
-  <div class="bubble" data-bind="bubble: $root.languageBubble">
+  <div class="bubble" data-bind="bubble: $root.scriptBubble">
     <p>
-      [% l('The language attribute should be set to the language used for the release title and track titles. It should not be set to the language the lyrics are written in, nor to the language used for other extra information on the cover.') %]
+      [% l('The {doc|script} attribute should be set to the script used for the release title and track titles.',
+           { doc => { href => doc_link('Release#Script'), target => '_blank' } }) %]
+    </p>
+    <p>
+      [% l('Please see the {doc|style guidelines} for more information.',
+           { doc => { href => doc_link('Style/Release#Language_and_script'), target => '_blank' } }) %]
     </p>
   </div>
 
-  <div class="bubble" data-bind="bubble: $root.scriptBubble">
+  <div class="bubble" data-bind="bubble: $root.dateBubble">
+    <!-- ko ifnot: target() && (target().hasAmazonDate() || target().hasJanuaryFirstDate()) -->
+      <p>
+        [% l('The {doc|date} describes when this particular release was made available.',
+             { doc => { href => doc_link('Release/Date'), target => '_blank' } }) %]
+      </p>
+      <p>
+        [% l('Please see the {doc|style guidelines} for more information.',
+             { doc => { href => doc_link('Style/Release#Date'), target => '_blank' } }) %]
+      </p>
+    <!-- /ko -->
+    <!-- ko if: target() && target().hasAmazonDate() -->
+      <p>
+        [% l('“1990-10-25” is the bogus date that Amazon gives to all releases for which they don’t know the actual date.
+              <strong>Please use this date only if you’re certain this date is correct.</strong>') %]
+      </p>
+    <!-- /ko -->
+    <!-- ko if: target() && target().hasJanuaryFirstDate() -->
+      <p>
+        [% l('If you do not know the month or day of release, please leave them empty.
```

Comment:
I wrapped some very long strings here. Hopefully that won't cause problems -- it seemed like it was already being done in other places.